README.md

```
@@ -0,0 +1,121 @@
+**원하는 굿즈만 쏙 덕후들을 하나로 잇는 공간**
+
+</br>
+
+## 🔢 목차
+[프로젝트 설명](#프로젝트-설명)</br>
+[서비스 목표](#서비스-목표)</br>
+[주요 기능](#주요-기능)</br>
+[아키텍쳐](#시스템-아키텍쳐)</br>
+[기술 및 아키텍쳐 선정](#기술-및-아키텍쳐-선정)</br>
+[컨벤션 규칙 및 브랜치 전략](#컨벤션-규칙-및-전략)</br>
+[파트원 사진](#파트원-사진)</br>
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)